libraries/botbuilder-spell-check-middleware/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@botbuildercommunity/spell-check-middleware",
+  "version": "0.1.0",
+  "description": "Bot Framework middleware component for the Cognitive Services Spell Check API",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "postinstall": "tsc -p .",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/BotBuilderCommunity/botbuilder-community-js.git"
+  },
+  "keywords": [
+    "chatbots",
+    "bot framework",
+    "bot builder",
+    "sentiment analysis",
+    "text analytics",
+    "cognitive services",
+    "middleware"
+  ],
+  "contributors": [
+    "Bot Builder Community",
+    "Stephan Bisser <stephan@bisser.at>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/BotBuilderCommunity/botbuilder-community-js/issues"
+  },
+  "homepage": "https://github.com/BotBuilderCommunity/botbuilder-community-js#readme",
+  "dependencies": {
+    "azure-cognitiveservices-spellcheck": "^1.0.1",
+    "botbuilder": "^4.1.3",
+    "ms-rest-azure": "^2.5.9",
```

Comment:
Are `ms-rest-azure` and `restify` dependencies for this library? I doesn't look like they are used in the middleware.

Reply:
The test bot in the test folder is using restify. Could probably make this a dev dependency. My next package is going to implement the test adapter and mocha for tests, so we can use that as a template for unit tests moving forward.

Reply:
Instead of moving `restify` do devDepencies, my advise would be to create a samples folder like the [dotNet repo](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet) has. And keep the test folder purely for unit tests.

Reply:
Once I put in `TestAdapter` unit testing along with mocha, we can follow that pattern for tests, and, yes, these test bots can then become samples.